source/features/set-default-repositories-type-to-sources.js

```diff
@@ -0,0 +1,16 @@
+import select from 'select-dom';
+
+export default function () {
+	// Get links set to repositories
+	for (const link of select.all('a[href*="tab=repositories"]')) {
```

Comment:
I think only one link needs to be changed: the tab

Reply:
Done